### PR TITLE
remaining german translations

### PR DESCRIPTION
**Pull request by MStraeten**
also revised some old translations to more common phrases
e.g. Kolorimetrisch (relativ/absolut) --> relativ/absolut Farbmetrisch

### CODE REVIEW

**Review by TurboGit — APPROVED** [review]
Thanks !

File: po/de.po [hunk 1]
```diff
@@ -4280,14 +4278,14 @@ msgid "later"
 msgstr "später"
 
 #: ../src/common/exif.cc:3980
-#, fuzzy, c-format
+#, c-format
 msgid "cannot read xmp file '%s': '%s'"
-msgstr "konnte xmp Datei „%s” nicht lesen: „%s\""
+msgstr "konnte xmp Datei '%s’ nicht lesen: ‘%s’"
```
Comment (rabauke) [inline po/de.po:4283]: Why this change? The typographically correct quotaion marks should be used here.
"konnte xmp Datei „%s” nicht lesen: „%s”"

File: po/de.po [hunk 2]
```diff
@@ -4280,14 +4278,14 @@ msgid "later"
...
 
 #: ../src/common/exif.cc:4032
-#, fuzzy, c-format
+#, c-format
 msgid "cannot write xmp file '%s': '%s'"
-msgstr "konnte xmp Datei „%s” nicht schreiben: „%s\""
+msgstr "konnte xmp Datei '%s' nicht schreiben: '%s'"
```
Comment (rabauke) [inline po/de.po:4288]: Same as above.

File: po/de.po [hunk 3]
```diff
@@ -7530,7 +7530,7 @@ msgstr "auto"
 #: ../src/gui/preferences.c:851 ../src/gui/preferences.c:957
 msgctxt "preferences"
 msgid "import..."
-msgstr "importieren"
+msgstr "importieren…"
```
Comment (rabauke) [inline po/de.po:7533]: The dots do not make any sense. Neither in the English original nor in the German translation. They just claim precious UI space.

File: po/de.po [hunk 4]
```diff
@@ -458,7 +458,7 @@ msgstr "nichtlinear Bradford"
 
 #: ../build/lib/darktable/plugins/introspection_channelmixerrgb.c:536
 msgid "XYZ "
-msgstr "XYZ"
+msgstr "XYZ "
```
Comment (rabauke) [inline po/de.po:461]: The additional space is unneeded. This holds for both, the  English as well as the German version.

File: po/de.po [hunk 5]
```diff
@@ -13456,6 +13459,8 @@ msgid ""
 "\n"
 "you should not touch values here !"
 msgstr ""
+"Internes Modul um Sensoreigenschaften zu einzustellen.\n"
```
Comment (rabauke) [inline po/de.po:13462]: WTF? Thie German sentence is completely unrelated to the English wording.

File: po/de.po [hunk 6]
```diff
@@ -16052,15 +16080,15 @@ msgstr "aus der Sammlung entfernen"
 
 #: ../src/libs/image.c:450
 msgid "move..."
-msgstr "verschieben"
+msgstr "verschieben…"
```
Comment (rabauke) [inline po/de.po:16083]: Again this dots thing. Why adding dots to some randomly chosen buttons?

File: po/de.po [hunk 7]
```diff
@@ -17849,7 +17880,7 @@ msgid_plural ""
 "%d images are assigned this tag!"
 msgstr[0] ""
 "Soll das Tag „%s“ wirklich gelöscht werden?\n"
-"%d Bild ist dieses Tag zugewiesen."
+"%d Bild ist dieses Tag zugewiesen!"
```
Comment (rabauke) [inline po/de.po:17883]: Why the exclamation point? This is not needed and inconsistent with the plural form.

File: po/de.po (hunk 5 above)
Comment (MStraeten) [reply po/de.po:13462]: Source:
```
 internal module to setup technical specificities of raw sensor.

you should not touch values here !
```
